### ISSUE TITLE
alexa-remote-routine, possible great unexpected feature (or bug)

### ISSUE THREAD

**Issue by juampe** [586837r/node-red-contrib-alexa-remote2#33]
Hi! I was testing the new release 3.0.5 with alexa-remote-routine

- Auth Cookie
- Service Host alexa.amazon.es
- Host amazon.es
- Languaje en-ES

While I was searching severals ways to pause alexa-remote-routtine with custom JSON (Yes I know that already are another dedicated node to do so [alexa-remote-echo]), I think that I found a way to feed arbitrary sentences to my echo, using JSON and custom

`{ "type": "music",  "payload": { "device": "Salón", "provider": "SPOTIFY", "search": "abre cadena cien en tune in", "duration": 0 }}`

`{ "type": "music",  "payload": { "device": "Salón", "provider": "SPOTIFY", "search": "pausa", "duration": 0 }}`

`{ "type": "music",  "payload": { "device": "Salón", "provider": "SPOTIFY", "search": "canta una canción", "duration": 0 }}`

As we see I injected arbitrary sentences in the **search** field, and alexa use it as a query. For me is what I was searching for my node-red automation project ,

Sure, I'm missing something couse I'm newbee using this node.
Keeping in mind that I tried to use this JSON without success.
```
{ "type": "routine", "payload": { "device":"Salón","routine": "pausa" }}
could not find routine: "pausa"
```

There are a node, dedicated to send arbitrary sentences to a echo device?
This way is acceptable to make a dedicated sentence node?

**Comment by 586837r** [comment]
Do you mean that using the music search works as if you were talking to the Echo?
That doesn't make any sense it should just play a song/album that you specified in "search".

There is currently no feature to send a string that works as if you were talking to the Echo. (that would be possible but would require bespoken.io)

The "routine"/ *Execute Routine* option executes routines that you have created in your Alexa app, This JSON would only work if you had created a routine that triggers when you say "pausa".

**Note:** using the *Custom* option is almost never necessary, you can use the *Music* option and still have the search or device be dynamic. The Routine node also has a *Stop* option that works similar to the *Pause* option of the Echo node.

**Comment by juampe** [comment]
> Do you mean that using the music search works as if you were talking to the Echo?
Yes, that is. It works as if I were talking to the Echo. And I think it is very useful to use alexa skill based automations. 
>That doesn't make any sense it should just play a song/album that you specified in "search".
Yes, I know, it's bizarre. But it seems to work.
`{ "type": "music", "payload": { "device": "Salón", "provider": "SPOTIFY", "search": "sing me a song", "duration": 0 }}`

>The Routine node also has a Stop option that works similar to the Pause option of the Echo node.

As far as I try it, the command only act as cancel skill, not like stop the music.

Te main reason to not to use this wonderfull node [alexa-remote-echo] as the developer expected is that i needed to simplify the flow, feeding "alexa commands" as payload, to custom.
`several vacation-timer [on|off string JSON]->json->alexa-remote-routine[custom as payload]`
I was able to search and play spotify music with this aproach [on vacation-timer], but no to pause or cancel it [off vacation.timer] until by mistake I altered the search command by **pausa** and **open bbc in tune in**... and... and here we are.

Good appreciation, I'll test later.
![imagen](https://user-images.githubusercontent.com/365683/62190622-16b33e00-b372-11e9-960a-6129cfa1128e.png)
